src/Validator/WhiteList.php

```diff
@@ -33,18 +33,25 @@ class WhiteList extends Validator
      */
     protected $strict;
 
+    /**
+     * @var string
+     */
+    protected $type;
```

Comment:
We should also add property types for all of them.

Reply:
Added property types in a bunch of places. 👍🏻